app/design/frontend/Diana/default-theme/web/css/source/_extend.less

```diff
@@ -67,6 +144,12 @@
     .identity-breakpoint {
         background-color: orange;
     }
+
+    .store-selector-main-wrapper {
+        .grid-of-stores {
+            grid-template-columns: 1fr;
```

Comment:
@Dianaeva Please, change grid to flex to avoid issues with browsers compatibility